sympy/parsing/c/c_parser.py

```diff
@@ -478,16 +488,29 @@ def transform_string_literal(self, node):
             pass
 
         def transform_character_literal(self, node):
-            #TODO: No string Type in AST
-            #type =
-            #try:
-            #    value = next(node.get_tokens()).spelling
-            #except (StopIteration, ValueError):
+            """Transformation function for character literal
+
+            Used to get the value of the given character literal.
+
+            Returns
+            =======
+
+            val : str
+                val contains the string value stored in the variable
+
+            Notes
+            =====
+
+            Only for cases where character is assigned to a integer value,
+            since character literal is not in sympy AST
+
+            """
+            try:
+               value = next(node.get_tokens()).spelling
+            except (StopIteration, ValueError):
                 # No tokens
-            #    value = node.literal
-            #val = [type, value]
-            #return val
-            pass
+               value = node.literal
+            return str(value[1])
```

Comment:
why does this have to be `value[1]`?
Wouldn't `value` have the character literal that can directly be returned?

Reply:
Actually, `node.literal` returns a str having value `'<char>'`, whereas we only want <char>. So, it is a sort of workaround.

Reply:
ok